package.json

```diff
@@ -26,7 +28,8 @@
     },
     "devDependencies": {
         "jasmine-node": "1.14.5",
-        "rewire": ">=2.1.3"
+        "rewire": ">=2.1.3",
+        "node-forge": "~0.6.18"
```

Comment:
I believe `prepublish` is run during `install` so  all required deps must be in `dependencies` (not in `devDependencies`), otherwise `npm install --production` will fail.